en/docs/learn/self-registration-and-account-confirmation.md

```diff
@@ -265,7 +242,21 @@ Next, you can try out self-registration.
         You can edit and customize the email template. For more information on
         how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails).
+
+!!! Note
+    If you wish to send a account unlocked email upon account confirmation, please do the following 
```

Comment:
As a practice, we generally avoid using "please" in our docs because we try to keep the tone neutral.

Reply:
Fixed: 34640c47b5c2100d1c6d914bcc9b6aeb406df775